extras/container/heketi-start.sh

```diff
@@ -70,6 +70,32 @@ if [[ $flock_status -ne 0 ]]; then
     fail "Database file is read-only"
 fi
 
+# this creates archival copies of the db if needed for disaster
+# recovery purposes
+if [[ "${HEKETI_DB_ARCHIVE_PATH}" ]]; then
+    HEKETI_DB_ARCHIVE_COUNT="${HEKETI_DB_ARCHIVE_COUNT:-3}"
+    # create an archive file for the previous copy of the db
+    # before the server starts up
+    mkdir -p "${HEKETI_DB_ARCHIVE_PATH}"
+    newname="heketi.db-archive-$(date +%Y-%m-%d.%s).gz"
+    gzip -9 -c "${HEKETI_PATH}/heketi.db" > "${newname}"
+    if [[ $? -ne 0 ]]; then
+        error "Unable to create archive"
```

Comment:
To alarm users slightly less, we should explain in log message that archive failure is expected in first run.

Reply:
Good point. I'll actually just skip the whole block if the db didn't previously exist and there's nothing to archive.